index.css

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap');
+
+.wrapper {
+  background-color: #1c1a19;
+  height: 100vh;
+  background-image: url('./images/bg-header-mobile.png');
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  background-size: 35%;
+}
+
+.tool-bar {
+  display: flex;
+  align-items: center;
+  color: #fff;
+  background-color: #3c3a39;
+  padding: 1rem 2rem;
+}
+
+.logo {
+  margin-right: auto;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 600;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 3rem 2rem 1rem 2rem;
+  margin: 2rem 1rem 0 1rem;
+}
+
+.head-main-title {
+  color: #ff6b00;
+  font-family: 'Crete Round', serif;
+  font-size: 5rem;
```

Comment:
Kindly reduce the `font-size` of the `.head-main-title` because your design does not match the Figma:

 - Your design:
   
![Cature](https://user-images.githubusercontent.com/95412943/212378620-4725d1ce-024e-4439-9f39-c4629e7aae59.PNG)
 - Figma design: 
 
![Captre](https://user-images.githubusercontent.com/95412943/212378663-69c1df40-441d-4f9e-928a-36e9060c4404.PNG)